src/lib/rxfill.ts

```diff
@@ -0,0 +1,40 @@
+import { Router, Request } from 'express';
+import { remsCaseCollection } from '../fhir/models';
+const router = Router();
+
+router.post('/ncpdp', async (req: Request) => {
```

Comment:
maybe rename this to ncpdp/script or something? I'm not sure if NCPDP has multiple different formats